controllers/reviews.controllers.js

```diff
@@ -0,0 +1,19 @@
+const { fetchReviewById } = require("../models/reviews.models");
+
+exports.getReviewById = (req, res, next) => {
+  const reviewId = req.params["review_id"];
+
+  fetchReviewById(reviewId)
+    .then((fetchedReview) => {
+      if (fetchedReview !== undefined) {
+        res.status(200).send({ review: fetchedReview });
+      } else {
+        res.status(404).send({
+          msg: "We couldn't find any reviews with that ID. Check your request and try again.",
+        });
+      }
```

Comment:
You could check this in the model instead of the controller. 

Before you send back to the controller, if the rows are empty, then you could return a promise.reject with the same status and message. If you do it that way, you can handle the error in your error handlers rather than having separate code here that you will likely repeat in every endpoint 😄